bip-hashrate-escrows/appendix-1.txt

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Why isn't this in the main BIP file?

Reply:
When describing anything, there is always a tradeoff between thoroughness and brevity.